packages/calypso-e2e/src/browser-manager.ts

```diff
@@ -114,11 +114,31 @@ export async function close(): Promise< void > {
 }
 
 /**
- * Given a page, this will clear the cookies for the context to which the page belongs.
+ * Given a page, this will clear all cookies, local storage and reset permissions for the
+ * Browser Context to which the page belongs.
  *
  * @param {Page} page Object representing a page launched by Playwright.
  * @returns {Promise<void>} No return value.
  */
 export async function clearCookies( page: Page ): Promise< void > {
```

Comment:
Optional: but what do you think about instead adding a wrapper function around this function, since this function now technically does more than just clearing cookies?

So this function, `clearCookies` can stay just clearing cookies, and then you make a wrapper function called `resetForNewUserLogin` that does the local storage wipe and calls into `clearCookies`?

I just don't want anyone to call this and miss the `localstorage` side effect if they don't know about it (and don't pay attention to the JSDoc header)

Reply:
I think it may be better to rename the method instead, since the `clearCookies` call is a one-liner (assuming the context is available) and I don't anticipate a situation that may require one but not the other.

What do you think about a name like `clearAuthenticationState` to be more in line with the original Selenium call (`ensureNotLoggedIntoSite`)?

Reply:
Oh, I like that! That name sounds nice!!